spec.html

```diff
@@ -130,9 +148,18 @@ <h1>Conformance</h1>
   <p>A conforming implementation of ECMAScript must not redefine any facilities that are not implementation-defined, implementation-approximated, or host-defined.</p>
   <p>A conforming implementation of ECMAScript may choose to implement or not implement <dfn>Normative Optional</dfn> subclauses. If any Normative Optional behaviour is implemented, all of the behaviour in the containing Normative Optional clause must be implemented. A Normative Optional clause is denoted in this specification with the words "Normative Optional" in a coloured box, as shown below.</p>
   <emu-clause id="sec-conformance.normative-optional" example normative-optional>
-    <h1>Example Clause Heading</h1>
+    <h1>Example Normative Optional Clause Heading</h1>
     <p>Example clause contents.</p>
   </emu-clause>
+  <p>A conforming implementation of ECMAScript must implement Legacy subclauses, unless they are also marked as Normative Optional. All of the language features and behaviours specified as “Legacy” have one or more undesirable characteristics and in the absence of legacy usage would be removed from this specification. However, the usage of these features by large numbers of existing web pages means that web browsers must continue to support them. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.</p>
```

Comment:
I don't think browsers should be called out here. After all, the point is that all implementations must implement legacy features.

```suggestion
  <p>A conforming implementation of ECMAScript must implement Legacy subclauses, unless they are also marked as Normative Optional. All of the language features and behaviours specified as “Legacy” have one or more undesirable characteristics. However, their continued usage in existing applications prevent their removal from this specification. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.</p>
```

Reply:
> their continued usage in existing applications prevent

s/prevent/prevents/

Reply:
Oops, good catch.

Reply:
Sounds good; “browsers” no longer makes sense now that it’s not in annex b.